src/App.css

```diff
@@ -1,42 +1,246 @@
-#root {
-  max-width: 1280px;
-  margin: 0 auto;
-  padding: 2rem;
+/* Reset */
```

Comment:
Well done implementing css to bring a unique look and feel to your application. Once thing to consider, this file is quite large. For your next project I would recommend breaking up your CSS files into files that correspond to component. i.e. App.css - for globally applied styles, TodoList.css - for your TodoList component, TodoListItem.css -> TodoListItem.jsx, etc